Formula/blogc.rb

```diff
@@ -0,0 +1,18 @@
+class Blogc < Formula
+  desc "Static blog compiler"
+  homepage "https://blogc.rgm.io"
+  url "https://github.com/blogc/blogc/releases/download/v0.12.0/blogc-#{version}.tar.bz2"
```

Comment:
This fails since you invoke `version` after `url`. In this case, skip `version` entirely and just put `0.12.0` in the `url`.